DEVELOPMENT_GUIDE.md

```diff
@@ -0,0 +1,221 @@
+# Development Guide
+
+This documentation is intended for developers to develop the schema, sandbox and proxies. It may be used by developers working within the NHS Digital organisation and outside contributors.
+
+## Table of Contents
+
+- [Development Guide](#development-guide)
+  - [Table of Contents](#table-of-contents)
+  - [Development](#development)
+    - [Requirements](#requirements)
+    - [Install](#install)
+      - [Updating hooks](#updating-hooks)
+    - [Environment Variables](#environment-variables)
+    - [Make commands](#make-commands)
+    - [Testing](#testing)
+    - [VS Code Plugins](#vs-code-plugins)
+    - [Emacs Plugins](#emacs-plugins)
+    - [Speccy](#speccy)
+    - [Caveats](#caveats)
+      - [Swagger UI](#swagger-ui)
+      - [Apigee Portal](#apigee-portal)
+    - [Platform setup](#platform-setup)
+    - [Detailed folder walk through](#detailed-folder-walk-through)
+      - [`/.github`:](#github)
+      - [`/azure`:](#azure)
+      - [`/proxies`:](#proxies)
+      - [`/sandbox`:](#sandbox)
+        - [Development](#development-1)
+      - [`/scripts`:](#scripts)
+      - [`/specification`:](#specification)
+      - [`/tests`:](#tests)
+      - [`Makefile`:](#makefile)
+      - [`ecs-proxies-containers.yml ` and `ecs-proxies-deploy.yml`:](#ecs-proxies-containersyml--and-ecs-proxies-deployyml)
+      - [`manifest_template.yml`:](#manifest_templateyml)
+      - [Package management:](#package-management)
+  - [Releasing a new schema version](#releasing-a-new-schema-version)
+
+## Development
+
+### Requirements
+
+-   GNU make
+-   nodejs + npm/yarn
+-   Python 3.8 + [poetry](https://github.com/python-poetry/poetry)
+-   Java 8+
+
+### Install
+
+```
+$ make install
+```
+
+#### Updating hooks
+
+You can install some pre-commit hooks to ensure you can't commit invalid spec changes by accident. These are also run
+in CI, but it's useful to run them locally too.
+
+```
+$ make install
```

Comment:
Is this intentionally the same command as the install section above?

Reply:
Removed the dedicated section as that install is used for testing. Added minor sections to each area that has it's own install